src/components/Button.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+
+const Button = props => (
+  <button type="button">
+    {props.name}
```

Comment:
kindly use destructing in here, and also let's practice a good naming convention.

example: 
```
const Button = ({buttonName}) => (
 <button type="button">
    {buttonName}
)
......
```

Once you have done this, I don't think you need to disable the rule at line 2 in this file